src/case-monitoring/sub-process.ts

```diff
@@ -140,21 +156,21 @@ function getWarningInfoAsHtml() {
                 <td>Maribel</td>
                 <td class="text-center">75%</td>
                 <td class="popover-action">
-                    <button class="btn btn-sm btn-success">Assign</button>
+                    <button id="assign0" class="btn btn-sm btn-success">Assign</button>
```

Comment:
**suggestion:** Can we have a more explicit id and use kebab case for consistency with the other ids? Something like `btn-assign-<number>`?

Reply:
fixed in d58e0cd125330f3dd4245ba564293f06a5d3d8cb